var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -66,16 +80,16 @@ def install(self, spec, prefix):
             make()
             make("install")
 
-        # The shared libraries are not installed correctly on Darwin; correct this
+        # The shared libraries are not installed correctly on Darwin:
         if (sys.platform == 'darwin') and ('+shared' in spec):
             fix_darwin_install_name(prefix.lib)
 
-
     def setup_dependent_package(self, module, dependent_spec):
         spec = self.spec
         lib_dsuffix = '.dylib' if sys.platform == 'darwin' else '.so'
```

Comment:
Since you are already changing the scalapack package, could you add the use of the `dso_suffix` here.